lib/api/controllers/securityController.js

```diff
@@ -449,7 +447,7 @@ class SecurityController {
     const pojoUser = request.input.body.content;
     pojoUser._id = request.input.resource._id || uuid.v4();
 
-    return Bluebird.coroutine(persistUserAndCredentials)(this.kuzzle, request, pojoUser);
+    return createGenerator(this.kuzzle, request, pojoUser);
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') Move the declaration of "createGenerator" before this usage. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)